languages/swift/exercises/concept/structs-and-classes/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+- `move(to:)` : `(Position) -> ()` - This is similar to `resize(to:)`, however, this method adjusts the _position_ of the window to the requested value, rather than the size. As with `resize` the new position cannot exceed certain limits. - The smallest position is 0 for both `x` and `y`. - The maximum position in either direction depends on the current size of the window; the edges cannot move past the edges of the screen. Values larger than these bounds will be clipped to the largest size they can take. E.g. if the window's size is at `x` = 250, `y` = 100 and a move to `x` = 600, `y` = 200 is requested, then the window would be moved to `x` = 550, `y` = 200 as the screen is not large enough in the `x` direction to fully accommodate the request.
+- `move(deltaX:deltaY)` : `(Int, Int) -> ()` - This is similar to `resize(deltaW:deltaH)`, however, this method adjusts the _position_ of the window to the requested value, rather than the size. As with `resize` the new position cannot exceed certain limits. - The smallest position is 0 for both `x` and `y`. - The maximum position in either direction depends on the current size and position of the window; the edges cannot move past the edges of the screen. Values larger than these bounds will be clipped to the largest size they can take. E.g. if the window's current size is `width` = 150, height = 150 and its current position is at `x` = 500, `y` = 450, and a move with `deltaX` = -600, `y` = 200 is requested, then the window would be moved to `x` = 0, `y` = 450.
+- `update(title:)` : `(String) -> ()` - This method sets the `title` property to the value of the string that was passed in.
+- `update(text:)` : `(String?) -> ()` - This method sets the `contents` property to the value of the optional string that was passed in.
+- `display()` : `() -> String` - This method returns a string describing the current state of the window. For example, if the window has the `title` "My First Window" with position: x = 10, y = 100; size: width = 200, height = 150; and contents: "I 😍 my window", it should return the string: `"My First Window\nPosition (10, 100), Size: (200 x 150)\nI 😍 my window\n"` - If `contents` is nil, the last line should read "[This window intentionally left blank]"
+
+## 4. Create some Windows
+
+Create two instances of windows and modify them via their methods as follows:
+
+- The first window should be given the title "Main Window", with a width of 400, a height of 300 and positioned at x = 100, y = 100. Its contents should be "This is the main window". Assign this instance to the name `mainWindow`.
+- The second window should be given the title of "Help Dialog", with a width and height of 100 each, positioned at x = 690 and y = 10. Its contents should be "Somebody called for help?". Assign this instance to the name `helpWindow`.
```

Comment:
Does this second window teach the student anything the first window doesn't?

Reply:
No. I've removed it.